whois-update/src/main/java/net/ripe/db/whois/update/handler/transformpipeline/TransformPipeline.java

```diff
@@ -0,0 +1,31 @@
+package net.ripe.db.whois.update.handler.transformpipeline;
+
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.Update;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TransformPipeline {
```

Comment:
I'm getting a bit muddled up between a PipelineTransformer and a TransformPipeline. Maybe use another name for one of them?